notebooks/README.md

```diff
@@ -0,0 +1,72 @@
+# Example Notebooks
+
+This directory contains examples about different main and different functionalities available in Fed-BioMed. It also contains a directory that points to tutorials published in the documentation site. The notebooks in the main directory does not contain details information, however, you can find more detailed examples in the `tutorials` directory.
+
+## Datasets Used in the examples
+
+Examples in the main directory uses different type of dataset. While some dataset are common among the examples, some of them are generated within the example notebooks. The instructions for loading generated datasets are available in the notebooks.   
+
+### Loading MNIST Dataset
+
+1. `{FEDBIOMED_DIR}/scripts/fedbiomed_run node dataset add`
+  * Select option 2 (default) to add MNIST to the node.
+  * Confirm default tags by hitting "y" and ENTER.
+  * Pick the folder where MNIST is downloaded.
+  * Data must have been added (if you get a warning saying that data must be unique is because it's been already added)
+
+2. Check that your data has been added by executing `{FEDBIOMED_DIR}/scripts/fedbiomed_run node dataset list`
+3. Run the node using `{FEDBIOMED_DIR}/scripts/fedbiomed_run node start`. Wait until you get `Starting task manager`. it means you are online.
+4. Following the same procedure, you can create a second node for client 2.
+
+
+### Loading Pseudo Adni Dataset
+
+It is necessary to previously configure a node (at least):
```

Comment:
Hi @sharkovsky, 
It is just a copy paste from notebooks.  I think that is why this sentence is left there. We can remove it because this part is only about adding a dataset which does not require configuring a node.